code/game/objects/structures/popout_cake.dm

```diff
@@ -43,7 +43,10 @@
 	var/string_pulled = 0
 
 /obj/structure/popout_cake/Destroy()
-	for(var/mob/living/L in locked_atoms + contents) //Release all mobs inside
```

Comment:
This change doesn't work, but I recently learned for x in null is fine syntax so I reckon we can revert to the old code